plotrix.Rmd

```diff
@@ -0,0 +1,124 @@
+# Plotrix for complex visualizations
+
+Alex Wan and Junyang Jiang
+
+```{r echo = FALSE, message=FALSE}
+  library(plotrix)
+```
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
```

Comment:
Please remove this chunk

Reply:
updated!

Reply:
Wait, your partner has already submitted this in a separate pull request https://github.com/jtr13/cc19/pull/77.
I don't think you need to submit it as well. Right? @bojum

Reply:
@kmsingh-git Good catch!! Only one submission needed for a group.

Reply:
so my partner and i actually did two separate things for our community contribution... this is different than the one he wrote